Gruntfile.js

```diff
@@ -14,7 +14,9 @@ const extract = require('jsxgettext-recursive');
 // use this if you want to recursively match all subfolders:
 // 'test/spec/**/*.js'
 module.exports = function (grunt) {
-  'use strict';
+  'use strict'
```

Comment:
I think this repo doesnt use semi in a lot of files, ill find out if there is a way to throw errors IFF semicolons are found :)